### ISSUE TITLE
Why are the cli cmds all panoptes prefixed?

### ISSUE THREAD

**Issue by camallen** [zooniverse/panoptes-cli#94]
It seems strange to tell users to run `panoptes` commands on their instead of say 'zooniverse'. To me `zooniverse project create "My Project" "This is a description of my project"` carries the intention better.

I think this is a leaky abstraction on how our systems are wired up. The configuration to talk to the panoptes API should take care of this (new systems will come and they can be configured accordingly). 

I understand the historical naming narrative but i'm not sure it makes much sense to use internal code words to describe how to interact with the Zooniverse platform.

**Comment by adammcmaster** [comment]
Yeah. This has been in the back of my mind since I originally started working on the CLI. I almost did call it "zooniverse" but in the end I went with "panoptes" since in principle someone could use it with a self-hosted instance of the API. I'm not really convinced that that was the right choice though, so I'd be interested in hearing what everyone else thinks.

**Comment by camallen** [comment]
I guess if you wanted both behaviour modes you could you expose 2 commands..
`zooniverse` that has sensible defaults available via configuration directives (i.e. what panoptes does now)
`panoptes` would require explicity API configuration directives and not linked to zooniverse (breaking change)
